packages/plugin-brotli/package.json

```diff
@@ -1,15 +1,14 @@
 {
   "name": "@pwa/plugin-brotli",
-  "version": "0.5.0",
+  "version": "0.5.1",
```

Comment:
General advice: don't increment versions in PRs. You don't know when or what else the maintainer(s) is planning for release. And in this case, it'd be wrong anyway. For two reasons, this PR is a breaking change, which would require this to be `0.6.0`